chat/src/main/java/com/github/twitch4j/chat/events/channel/DirectCheerEvent.java

```diff
@@ -78,60 +38,51 @@ public class DirectCheerEvent extends CheerEvent {
     public DirectCheerEvent(IRCMessageEvent event) {
         this(
             event,
-            event.getTagValue("msg-param-amount").map(Integer::parseInt).orElse(0),
-            event.getTagValue("system-msg").map(String::trim).orElse("")
+            event.getTagValue("msg-param-amount").map(Long::parseLong).orElse(0L),
+            event.getTagValue("msg-param-exponent").map(Integer::parseInt).orElse(2),
+            event.getTagValue("msg-param-currency").orElse("USD")
         );
     }
 
-    DirectCheerEvent(IRCMessageEvent event, Integer amount, @NotNull String systemMessage) {
-        super(event, event.getChannel(), event.getUser(), event.getMessage().orElse(""), Math.round(amount * CREATOR_REVENUE_SPLIT), event.getSubscriberMonths().orElse(0), event.getSubscriptionTier().orElse(0), event.getFlags());
-        this.amount = amount;
-        this.systemMessage = systemMessage;
+    DirectCheerEvent(IRCMessageEvent event, long amount, int exponent, String currency) {
+        super(event,
+            event.getChannel(),
+            event.getUser(),
+            event.getMessage().orElse(""),
+            "USD".equalsIgnoreCase(currency) ? Math.round(amount * CREATOR_REVENUE_SPLIT) : 0,
```

Comment:
The `#getBits` method can be deprecated now to discourage use as we don't track currency fluctuations or payment processor fees

If bits as a whole are killed by twitch (increasingly likely), the whole super class can be deprecated